docs/usage/self-hosted-experimental.md

```diff
@@ -39,6 +39,10 @@ You can set the config file Renovate should read with the `RENOVATE_CONFIG_FILE`
 
 The process that runs Renovate must have the correct permissions to delete the config file.
 
+## `RENOVATE_X_DISABLE_DOCKER_HUB_LABELS`
```

Comment:
```suggestion
## `RENOVATE_X_DOCKER_HUB_DISABLE_LABEL_LOOKUP`
```
I prefer this naming to correlate with other options.

Reply:
I've updated the variable name accordingly.